jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Attributes.java

```diff
@@ -135,6 +136,34 @@ static Attributes unwrap(Attributes attributes)
         return attributes;
     }
 
+    static int hashCode(Attributes attributes)
+    {
+        int hash = 113;
+        for (String name : attributes.getAttributeNameSet())
+            hash += name.hashCode() ^ attributes.getAttribute(name).hashCode();
+        return hash;
+    }
```

Comment:
yeah.... but no.  Already tried that and java doesn't let you have a default method that overrides a method in `Object`.

Reply:
Ok, a surprise you can't override.

Then please override `hashCode()` and `equals()` and make them `abstract` (it can be done), so it's clear that implementation must call these `static` methods.